packages/graphql/tests/schema/directives/settable.test.ts

```diff
@@ -0,0 +1,1476 @@
+                  title_SHORTEST_LTE: Int @deprecated(reason: \\"Please use the explicit _LENGTH version for string aggregation.\\")
+                }
+
+                type ActorActedInRelationship {
+                  cursor: String!
+                  node: Movie!
+                }
+
+                input ActorActedInUpdateConnectionInput {
+                  node: MovieUpdateInput
+                }
+
+                input ActorActedInUpdateFieldInput {
+                  connect: [ActorActedInConnectFieldInput!]
+                  create: [ActorActedInCreateFieldInput!]
+                  delete: [ActorActedInDeleteFieldInput!]
+                  disconnect: [ActorActedInDisconnectFieldInput!]
+                  update: ActorActedInUpdateConnectionInput
+                  where: ActorActedInConnectionWhere
+                }
+
+                type ActorAggregateSelection {
+                  count: Int!
+                  name: StringAggregateSelectionNonNullable!
+                }
+
+                input ActorConnectInput {
+                  actedIn: [ActorActedInConnectFieldInput!]
+                }
+
+                input ActorCreateInput {
+                  actedIn: ActorActedInFieldInput
+                  name: String!
+                }
+
+                input ActorDeleteInput {
+                  actedIn: [ActorActedInDeleteFieldInput!]
+                }
+
+                input ActorDisconnectInput {
+                  actedIn: [ActorActedInDisconnectFieldInput!]
+                }
+
+                type ActorEdge {
+                  cursor: String!
+                  node: Actor!
+                }
+
+                type ActorMovieActedInAggregationSelection {
+                  count: Int!
+                  node: ActorMovieActedInNodeAggregateSelection
+                }
+
+                type ActorMovieActedInNodeAggregateSelection {
+                  description: StringAggregateSelectionNullable!
+                  title: StringAggregateSelectionNonNullable!
+                }
+
+                input ActorOptions {
+                  limit: Int
+                  offset: Int
+                  \\"\\"\\"
+                  Specify one or more ActorSort objects to sort Actors by. The sorts will be applied in the order in which they are arranged in the array.
+                  \\"\\"\\"
+                  sort: [ActorSort!]
+                }
+
+                input ActorRelationInput {
+                  actedIn: [ActorActedInCreateFieldInput!]
+                }
+
+                \\"\\"\\"
+                Fields to sort Actors by. The order in which sorts are applied is not guaranteed when specifying many fields in one ActorSort object.
+                \\"\\"\\"
+                input ActorSort {
+                  name: SortDirection
+                }
+
+                input ActorUpdateInput {
+                  actedIn: [ActorActedInUpdateFieldInput!]
```

Comment:
Sorry, I probably told you to change the wrong line, this is the field that should be removed by `onUpdate: false`. Currently, the schema here doesn't change if you set `onUpdate: true`

```suggestion
```